core/domain/exp_services_test.py

```diff
@@ -6292,6 +6835,15 @@ def test_draft_version_valid_when_no_draft_exists(self):
         self.assertFalse(exp_services.is_version_of_draft_valid(
             self.EXP_ID3, exp_user_data.draft_change_list_exp_version))
 
+    def test_create_or_update_draft_when_by_voice_artist(self):
+        with self.assertRaisesRegex(
+            utils.ValidationError,
+            'Voice artist does not have permission to make some '
+            'changes in the change list.'):
+            exp_services.create_or_update_draft(
+            self.EXP_ID1, self.USER_ID, self.NEW_CHANGELIST, 5,
+            self.NEWER_DATETIME, True)
```

Comment:
These lines need to be indented one level

Reply:
fixed

Reply:
It doesn't look like this is fixed yet?